### PR TITLE
Add a getting started doc for swck

### PR DESCRIPTION
**Pull request by dashanji**
Add a document to help users quickly get started with swck, including the following aspects:
- Create a local kubernetes cluster.
- Build the docker image of operator and deploy it on the kubernetes cluster.
- Deploy the skywalking components: OAPServer and UI.
- Use the injector to inject the java agent to the application pod.
- Access the UI and get the metric info.

### CODE REVIEW

**Comment by wu-sheng** [comment]
The screenshot should be submitted to website repo, https://github.com/apache/skywalking-website/tree/master/static/doc-graph

And you could use the https://skywalking.apache.org/doc-graph/ in the source code repository.

**Comment by wu-sheng** [comment]
And that screenshot is 238k, please make sure it has been compressed properly.

**Comment by dashanji** [comment]
Thanks for the detail，I will submit the image later.

**Comment by wu-sheng** [comment]
@hanahmily Please take a look.

File: docs/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+    app: demo
+EOF
+```
+
+Check the status of the application pod and make
+sure the java agent is injected into the application pod.
+
+
+```shell
+$ kubectl get pod -n skywalking-system -l app=demo -ojsonpath='{.items[0].spec.initContainers[0]}'
```
Comment (hanahmily) [inline docs/getting-started.md:274]: How to check the agent's config and logs. Both of them is useful to debug relevant issues.

File: docs/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+
+### Prerequisites
+- [docker](https://docs.docker.com/get-docker/)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
+- [swctl](https://github.com/apache/skywalking-cli?tab=readme-ov-file#install)
+
+### Step1: Create a kubernetes cluster locally using kind
+
+> Note: If you have a kubernetes cluster already, you can skip this step.
```
Comment (hanahmily) [inline docs/getting-started.md:13]: What's the required version?

File: docs/getting-started.md (hunk 1 above)
Comment (dashanji) [reply docs/getting-started.md:274]: Make sense.

File: docs/getting-started.md (hunk 2 above)
Comment (dashanji) [reply docs/getting-started.md]: Updated. It should be more than v1.21.10

**Comment by dashanji** [comment]
@hanahmily I have addressed these comments. Could you please take another review?

**Comment by wu-sheng** [comment]
@hanahmily Could you recheck on this?

**Comment by wu-sheng** [comment]
@hanahmily friendly ping

**Comment by wu-sheng** [comment]
@dashanji Please fix CI. And should we release a new version after this?

**Comment by dashanji** [comment]
> @dashanji Please fix CI. And should we release a new version after this?

Yes, That's what I planned.

**Comment by wu-sheng** [comment]
Could you upgrade go to 1.21.8+? It seems to have some CVEs for tor the current version.

**Comment by dashanji** [comment]
> Could you upgrade go to 1.21.8+? It seems to have some CVEs for tor the current version.

Okay.

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**